### DISCUSSION TITLE
Any plan to add support for RTL8821CE?

### DISCUSSION THREAD

**Discussion by sugarylettuce49** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#347] [Ideas]
Is there any plan to add support for RTL8821CE? This would be really great!

**Comment by chris1111** [comment]
There is no source for this project, All wifi companies have abandoned it as well as Realtek the main builder.
There is [no source for the Driver](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/discussions/209)

**Reply by shotta2024** [reply]
Have you attempted any work on it?

**Comment by sugarylettuce49** [comment]
I see, that’s very unfortunate..

**Comment by shotta2024** [comment]
Is this the same case for Realtek 8822CE?
